### DISCUSSION TITLE
애니메이션이 제대로 작동하지 않습니다.

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1149] [5. Q&A]
캐릭터에 애니메이션을 재생하고 싶은데,  실제 애니메이션과 움직임이 다릅니다.
![image](https://user-images.githubusercontent.com/59792911/183593401-ea56c1fd-ad40-4a15-98f0-fdf8474bdc58.png)

![image](https://user-images.githubusercontent.com/59792911/183593487-2e89d23f-ff26-4b73-b001-110343ae266f.png)

믹사모에서 Unity전용으로 받은 애셋입니다.

**Comment by ZdevMonZ** [comment]
갖고오신 파일에 몇가지 설정을 해야 되더라구요.
https://github.com/naverz/zepeto-studio-kor/discussions/538 요거랑
https://youtu.be/9H0aJhKSlEQ 요 유튜브 영상이 도움됐었어요.
